jwala-integration-test/src/test/java/com/cerner/jwala/ui/selenium/steps/configuration/HandleResourceRunSteps.java

```diff
@@ -144,6 +144,13 @@ public void selectResourceCheckBox(String resourceFile) {
 
     @Then("^I confirm delete a resource popup$")
     public void deleteResource() {
+        jwalaUi.isElementExists(By.xpath("//*[contains(text(), 'Are you sure you want to delete the selected resource template(s) ?')]"));
```

Comment:
What was the fix on the "weird error" ? Is it using "//*..." instead of "//div..." ? The thing is I noticed that the code you added in HotDeployRunSteps.java at line 80 uses "//div". I guess it works since you used it. I avoid using asterisk since I want my query to be more concise.